projects/managed-hubs/incidents.md

```diff
@@ -0,0 +1,120 @@
+# Incident response
+
+
+When an {term}`Incident` is declared, we trigger a special response in order to ensure that it is resolved quickly.
+This section describes our incident response process, major roles and terminology, and what to expect.[^incident-refs]
+
+[^incident-refs]: The [PagerDuty Incident Response Guide](https://response.pagerduty.com/) and the [Google SRE Incident response guide](https://sre.google/workbook/incident-response/) inspired much of the content on this page.
+
+:::{admonition} In Beta!
+:class: warning
+We are currently working out our Incident Response process.
+The content on this page might change over time, and we welcome suggested changes and pull requests!
+:::
+
+## Roles and team structure
+
+An {term}`Incident Response Team` is formed when an {term}`Incident` has been declared.
+The goal of the Incident Response Team is to collectively resolve incidents.
+
+An Incident Response Team is generally made up of:
+
+- An {term}`Incident Commander`
+- The {term}`Support Steward`s
+- One or more {term}`Subject Matter Expert`s (SMEs)
+
+```{glossary}
+Incident Response Team
+  The group of roles that collectively understand, plan, resolve, and communicate our actions around an {term}`Incident`. The people in these roles may change in a fluid manner, and one person may serve in multiple roles. A rough way to approximate this team is "the people that have communicated in internal and external channels to resolve an incident."
+
+Incident Commander
+  The Incident Commander has the authority to plan and delegate action to others on the {term}`Incident Response Team`. They are **not expected** to take actions themselves. Their goal is to help the team make consistent and deliberate progress towards resolving an incident. They are the {term}`Source of Truth` about the current state and action plan surrounding an incident. 
+
+Subject Matter Expert
+  A member on the {term}`Incident Response Team` with expertise in an area of relevance to an Incident. SMEs have a variety of backgrounds and abilities, and they should be pulled in to the Response Team as-needed by the {term}`Incident Commander`. Their goal is to take actions as-directed by the {term}`Incident Commander` to resolve an incident.
+```
+
+## Communication channels
+
+### External communication
+
+- The {term}`Support Steward` team acts as the primary point of communication with external stakeholders like the {term}`Community Representative`s.
```

Comment:
IMO, specifically for incidents, this should be *optional*. This adds an additional load for the support steward and the incident commander (if they are different people). The IC can request the support steward to act as this if necessary, but otherwise I think we should *not* require this of the support steward. They should be able to delegate an incident to the incident commander, and then by default continue with their existing role. I think if the IC is the source of truth, they should by default be the person who communicates too - otherwise we're adding an entirely new person to this chain, and that is often extremely frustrating during an incident process for everyone involved.

So my suggestion is that the IC can ask someone else to be the point of contact (support steward or someone else) if they so choose to, but that is not the default.

Reply:
Your reasoning makes sense to me - one concern I have is that the incident commander needs to spend their cycles on resolving the incident, not necessarily also communicating it externally. Maybe the answer is to say the incident commander does this by default, but if they must log-off or are otherwise overwhelmed, they should delegate another team member to provide external communication?

Reply:
Exactly right! The default is they do it but if in their judgement delegating it is the right thing - if the extra communication overhead is worth it (as it often is) they can. They just delegate it to someone, not necessarily the support steward.

Reply:
As long as the IC presents itself (or it is actually introduced by the support steward) to the ticket submitter, I think it makes sense NOW to make this optional. I also like the idea to delegate the communication to another one who is not necessarily the support steward (because we do not have a lot of people in support to address other tickets).

Why do I highlight the NOW word? Because when we get a dedicated support team, there should be a clear separation of boxes, IMHO. The support team should be handling the communication with the ticket submitters because they are trained and specialized to interact with people in stress looking for answers. An IC coming from the eng team is not well prepared for that interaction... and that might be a source of issues.

Reply:
yeah - thinking to the [Incident Commander docs from pagerduty](https://response.pagerduty.com/before/different_roles/) I think this is the "external liason" role. They also have a dedicated person/team to do that work. In our case, I think there are two things I worry about there:

- We don't have the staffing capacity for this now, but maybe this will change in the future as @damianavila suggests.
- If people are not awake at the same time, we pay a big communication penalty when we have bottlenecks of information. If one person must be the one to communicate externally, and that person just went to sleep, then it means no communication can occur until they return to work. This feels like a stressful situation given that we don't have the staffing to ensure seamless handoffs between time-zones all the time.

Reply:
Yep, this is why I think this is fine for now but we should change it in the future when we have enough capacity.

Reply:
OK I've updated this so that the Incident Commander is the communicator by default, and may delegate if they wish.